src/main/webapp/i18n/de/textSubmission.json

```diff
@@ -15,7 +15,8 @@
             "detail": {
                 "title": "Text Submission"
             },
-            "text": "Text"
+            "text": "Text",
+            "view": "Text Submission ansehen"
```

Comment:
Submission --> Einreichung

(please also change all other wrong values in this file)

Reply:
See thread above.